stacks-common/build.rs

```diff
@@ -0,0 +1,43 @@
+use std::path::Path;
+use std::{env, fs};
+
+use toml::Value;
+
+fn main() {
+    let toml_content =
+        fs::read_to_string("../versions.toml").expect("Failed to read versions.toml");
```

Comment:
Can you use `cargo_workspace()` from `stacks_common::util` instead of relative paths?

```suggestion
    let toml_file = cargo_workspace("versions.toml");
    let toml_content =
        fs::read_to_string(toml_file).expect("Failed to read versions.toml");
```

Reply:
Hm, unfortunately this won't work, as you can't import code from within the package from that package's build script.